src/System.Device.Gpio/System/Device/Pwm/Channels/UnixPwmChannel.Linux.cs

```diff
@@ -184,6 +195,10 @@ public override void Stop()
 
         protected override void Dispose(bool disposing)
         {
+            _dutyCycleWriter?.Dispose();
```

Comment:
Will this also call Close on the FileStream?

Reply:
yes